Source/Core/DolphinQt/Debugger/MemoryViewWidget.cpp

```diff
@@ -324,10 +342,23 @@ void MemoryViewWidget::mousePressEvent(QMouseEvent* event)
   switch (event->button())
   {
   case Qt::LeftButton:
-    if (column(item) == 0)
+
+    if (event->modifiers() & Qt::ShiftModifier)  // Shift click to load Address into box
```

Comment:
This comment isn't necessary. The code itself literally indicates this, along with the face it's within the mouse click handler.